packages/experiments/src/components/Shimmer/Shimmer.styles.ts

```diff
@@ -0,0 +1,79 @@
+import { IShimmerStyleProps, IShimmerStyles } from './Shimmer.types';
+import {
+  keyframes,
+  DefaultPalette
+} from '../../Styling';
+
+export function getStyles(props: IShimmerStyleProps): IShimmerStyles {
+  const {
+    width,
+    maxHeight,
+    isDataLoaded
+  } = props;
+
+  const shimmerAnimation: string = keyframes({
+    '0%': {
+      backgroundPosition: '-900%'
```

Comment:
How did we come with these two values? Do they scale with large width? Should we have a const variable?